dotnetv3/dynamodb/scenarios/DynamoDB_Basics/DynamoDB_Basics_ScenarioTests/DynamoDbMethodsTests.cs

```diff
@@ -37,26 +35,44 @@ public async Task PutItemAsyncTest()
             Assert.True(success, "Couldn't add the movie.");
         }
 
-        [Fact, TestPriority(3)]
+        [Fact()]
+        [Order(3)]
         public async Task UpdateItemAsyncTest()
         {
             var updateMovie = new Movie
             {
-                Title = "Now You See Me",
-                Year = 2013,
+                Title = "North by Northwest",
+                Year = 1959,
             };
 
             var updateMovieInfo = new MovieInfo
             {
-                Plot = "An FBI agent and an Interpol detective track a team of illusionists who pull off bank heists during their performances and share the wealth with the audience.",
+                Plot = "A classic Hitchcock Thriller.",
                 Rank = 5,
             };
 
             var success = await DynamoDbMethods.UpdateItemAsync(client, updateMovie, updateMovieInfo, tableName);
             Assert.True(success, $"Couldn't update {updateMovie.Title}.");
         }
 
-        [Fact, TestPriority(4)]
+        [Fact()]
+        [Order(4)]
+        public void ImportMoviesWithBadFileNameShouldReturnNullTest()
+        {
+            var movies = DynamoDbMethods.ImportMovies(testMovieFileName);
```

Comment:
It would be nice to have the variable name match the expected behavior here. Since we're passing bad data to ImportMovies (per the method name), a better variable name would be something like `badMovieFile` instead of `testMovieFileName`. Then we know just by reading that line in the test that it shouldn't succeed. I would also suggest changing the content of that variable so something like "notarealdatafile" instead of "testmoviedata.json" so someone doesn't think "testmoviedata.json" is supposed to be a real file and is missing.

Reply:
Changed as suggested.